docs/developer-guide/fetching-data.md

```diff
@@ -13,6 +13,8 @@ The following steps must be completed before you can pull data from an evironmen
 2. On the `#ds-etna` or `#ds-etna-dev` Slack channels, request for someone to add you to the Etna project.
 3. Once you have access, [generate an API token](https://docs.platform.sh/development/cli/api-tokens.html#get-a-token) for your account (The name **Local CLI** will do nicely), and add it your local `.env` file as `PLATFORMSH_CLI_TOKEN`.
 
+Developers from external agencies may not be able to register for a platform.sh account. In this scenario, ask a developer from The National Archives to share their API key.
```

Comment:
```suggestion
** Developers from external agencies may not be able to register for a platform.sh account. In this scenario, ask a developer from The National Archives to share their API key.
```

Suggested adding an asterisk to this so it's more obviously related to the above points

Reply:
same query here - I might have missed a subtlety of markdown, but thought the `**` needed a closing pair to make the text between bold

Reply:
@janehughes it would be nice to get this one merged - just wondering if you had a response to my queries above?

Reply:
And this was meant to be the footnote text :)